pkgs/applications/networking/browsers/brave/default.nix

```diff
@@ -87,8 +87,11 @@ let
   enableFeatures = optionals enableVideoAcceleration [ "VaapiVideoDecoder" "VaapiVideoEncoder" ]
     ++ optional enableVulkan "Vulkan";
 
-    # The feature disable is needed for VAAPI to work correctly: https://github.com/brave/brave-browser/issues/20935
-  disableFeatures = optional enableVideoAcceleration "UseChromeOSDirectVideoDecoder";
+  disableFeatures = [
+    "OutdatedBuildDetector" # disable automatic updates
+  ]
+  # The feature disable is needed for VAAPI to work correctly: https://github.com/brave/brave-browser/issues/20935
+  ++ optional enableVideoAcceleration "UseChromeOSDirectVideoDecoder";
```

Comment:
```suggestion
  ++ optionals enableVideoAcceleration [ "UseChromeOSDirectVideoDecoder" ];
```

Reply:
While I understand your perspective, I find myself unable to fully endorse it. For beginners, the use of `lib.optional` often leads to confusion, particularly with the incorrect syntax (ex: `lib.optional predicate [ item1 item2 ]`), resulting in items being doubly wrapped within two nested lists. This frequent misunderstanding adds to the complexity and is a common point of contention in reviews.

Moreover, given that the attribute you're updating is in its plural form, it seems logical to me to employ `lib.optionals` in this context.

Additionally, there is also some relevant discussion on this topic on Discourse: [https://discourse.nixos.org/t/lib-optional-considered-harmful/7081](https://discourse.nixos.org/t/lib-optional-considered-harmful/7081).

I advocate for the use of `lib.optionals` when appropriate, even though `lib.optional` might technically suffice. This stance is based on striving for clarity and reducing common errors, especially for those new to Nix.

Reply:
I see your point, but I believe @lheckemann perspective was pretty reasonable:

> What I’m not keen on is people changing this without it being a documented policy, because otherwise we can end up with it being changed back and forth, wasting people’s time and introducing meaningless code churn.

(in the discussion you cited)

Reply:
This is the reason why I've just commented here and not set it as required changes.